ionoscloud/resource_k8s_cluster.go

```diff
@@ -632,7 +633,9 @@ func k8sClusterReady(ctx context.Context, client *ionoscloud.APIClient, d *schem
 	if utils.IsStateFailed(*resource.Metadata.State) {
 		return false, fmt.Errorf("error while checking if k8s cluster is ready %s, state %s", *resource.Id, *resource.Metadata.State)
 	}
-	return *resource.Metadata.State == "ACTIVE", nil
+	log.Printf("[INFO] k8sClusterReady k8s cluster state: %s", *resource.Metadata.State)
```

Comment:
@cristiGuranIonos 
```suggestion
	log.Printf("[INFO] k8s cluster state: %s", *resource.Metadata.State)
```

or

```suggestion
	log.Printf("[INFO] current k8s cluster state: %s", *resource.Metadata.State)
```